### PR TITLE
Publisher type: translator

### LINKED ISSUE #202: Add translator as option for type of publisher
We introduced the conformance target CSAF translator. However, the current publisher type model requires them to have the value `other` for `/document/publisher/type`.

I suggest to add the option `translator` and make it mandatory for translators.

### PR DESCRIPTION
**Pull request by tschmidtb51**
- resolves #202
- add publisher type "translator"
- change prose
- add constraints regarding "source_lang" and conformance target

### CODE REVIEW

**Comment by tschmidtb51** [comment]
I think the TC needs to vote on that.

@santosomar: Please add it to the agenda.

File: csaf_2.0/json_schema/csaf_json_schema.json [hunk 1]
```diff
@@ -591,6 +591,7 @@
                 "coordinator",
                 "discoverer",
                 "other",
+                "translator",
```
Comment (sthagen) [inline csaf_2.0/json_schema/csaf_json_schema.json:594]: Yes!

**Review by sthagen — APPROVED** [review]
LGTM.

**Review by mprpic — APPROVED** [review]

**Comment by santosomar** [comment]
This pull request was discussed and the motion to accept the changes were approved during the CSAF TC meeting on 2021-04-14.

**Review by sthagen — COMMENTED** [review]
LGTM

**Pull request merged**